debian/control

```diff
@@ -3,6 +3,7 @@ Architecture: amd64
 Depends: binutils,
          libjemalloc1 | libjemalloc2,
          libsnappy1 | libsnappy1v5,
+         liblz4-1, liblz4-dev,
```

Comment:
We don't need liblz4-dev in a user's runtime.

Reply:
Fixed